code/modules/lore_codex/lore_data/main.dm

```diff
@@ -0,0 +1,32 @@
+/datum/lore/codex/category/main_vir_lore // The top-level categories for the Vir book
```

Comment:
I don't like ANY of the changes to the lore_codex, and insist we start building our own as a replacement.